sdk/core/core/src/az_aad.c

```diff
@@ -84,13 +84,13 @@ AZ_NODISCARD az_result _az_aad_build_body(
   return AZ_OK;
 }
 
-AZ_NODISCARD az_result _az_aad_request_token(_az_http_request* ref_request, _az_token* out_token)
+AZ_NODISCARD az_result _az_aad_request_token(_az_http_request* request, _az_token* out_token)
 {
   // FIXME: If you uncomment the line below, we'll start getting HTTP 400 Bad Request instead of 200
   // OK. I suspect, it is because there's a bug in the code that adds headers. Could be something
   // else, of course.
   /*AZ_RETURN_IF_FAILED(az_http_request_append_header(
-      ref_request,
```

Comment:
I think we should use the `ref_request` naming

Reply:
I'm fine with this but we need to go through the whole code base and fix 'ref_' everywhere.
I think I fixed a lot of 'out_'s so I think we're OK with this.